mobile/lib/src/app.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
```

Comment:
Move `lib/src` content to `lib/`

Reply:
### lib directory

    As you might expect, the library code lives under the lib directory and is public to other packages. You can create any hierarchy under lib, as needed. By convention, implementation code is placed under lib/src. Code under lib/src is considered private; other packages should never need to import src/.... To make APIs under lib/src public, you can export lib/src files from a file that’s directly under lib.

https://dart.dev/guides/libraries/create-library-packages

Reply:
TIL